index.html

```diff
@@ -19,24 +19,24 @@
 <body>
   <div class="container">
     <div class="heading">
-      <h1 class="title">2048</h1>
+      <h1 class="title"><strong><span class="gurmukhi">2048</span>!</strong></h1>
```

Comment:
The CSS styles already make the title bold and use Gurmukhi font, so these tags aren't needed :)